pages.es/common/more.md

```diff
@@ -0,0 +1,19 @@
+# more
+
+> Abre un archivo para lectura interactiva, permitiendo desplazar y buscar (Solamente en una dirección, adelante).
+
+- Abre un archivo:
+
+`more {{archivo}}`
+
+- Página abajo:
+
+`<Espacio>`
+
+- Busca una termino (La tecla `n` se mueve al siguiente resultado):
+
+`/{{termino de busqueda}}`
```

Comment:
We use variables in `snake_case`, if you use `cadena` in the example description, I think this should be:
```suggestion
`/{{cadena}}`
```